dtc-systems/random_walk.py

```diff
@@ -0,0 +1,96 @@
+"""
+  Follow Path - ver0 static, ver1 dynamically updated via msg "path'
+"""
+import math
+
+from osgar.lib.route import Route as GPSRoute, DummyConvertor
+from osgar.lib.line import Line
+from osgar.lib.mathex import normalizeAnglePIPI
+from osgar.node import Node
+from osgar.bus import BusShutdownException
+from osgar.followme import EmergencyStopException
+
+
+class Route(GPSRoute):
+    """
+    Override GPS defaults to normal planar case
+    """
+    def __init__(self, pts=[]):
+        super().__init__(pts, conv=DummyConvertor(), isLoop=False)
+
+
+class RandomWalk(Node):
```

Comment:
Something similar is already in osgar? Im just curious.

Reply:
? It is possible - this is a bit "work in progress"